services/api/service.go

```diff
@@ -919,7 +922,7 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 	defer func() {
 		submissionEntry, err := api.db.SaveBuilderBlockSubmission(payload, simErr, isMostProfitableBlock)
 		if err != nil {
-			log.WithError(err).Error("saving builder block submission to database failed")
+			log.WithError(err).WithField("bidTrace", payload.Message).Error("saving builder block submission to database failed")
```

Comment:
yeah good thought, although i think we'd primarily be interested in backfilling the delivered payloads, rather than all builder submissions 🤔 might not hurt to log that too though

Reply:
added